src/test/java/com/google/javascript/clutz/goog_promise_with_platform.js

```diff
@@ -0,0 +1,131 @@
+//!! goog.Promise, unlike other implementations of Promise, has 2 template params
+//!! this test ensures that the specially handled TTE methods work for it as well
```

Comment:
closure libs are on github - add a reference here for that.

Reply:
Done.